config/locales/sv.yml

```diff
@@ -632,6 +632,16 @@ sv:
       member: *member
       pay_membership_tooltip: När du betalar din medlemsavgift förlängs ditt medlemskap i ett år.
       membership_expire_date_tooltip: Efter detta datum går perioden för ditt medlemskap ut.
+      proof_of_membership: Bevis på medlemskap
+      proof_of_membership_describe: Den här bilden är ditt bevis på medlemskap i SHF.
```

Comment:
/* Remember we are moving away from the use of "SHF" */ 

Detta är ditt behörighetsbevis.